app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -47,7 +58,33 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
               className="w-full flex-grow-0"
               label="Description"
               name="description"
-              rows={8}
+              rows={1}
+            />
+            <Select
+              isClearable
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label="Assigned Contact"
+              name="contact"
+              placeholder="Select Role"
+              options={CONTACT_NAMES.map(contactName => ({
+                label: contactName,
+                value: contactName,
+              }))}
+            />
+            <Select
+              isClearable
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label="Tag"
+              name="tag"
+              placeholder="Select Tag"
+              options={TAGS.map(tag => ({
+                value: tag,
+                label: tag,
+              }))}
```

Comment:
Why exactly is that in your business logic you are not allowing a `note` to have multiple tags?